CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 6.0.0-beta.1
+### Breaking Changes
+* Purchase.Builder (#825) (#850) via beylmk (@beylmk)
```

Comment:
maybe we should call out that subscription option purchase methods were removed? i know this is auto-generated, i probably makred some wrong..